service/informer/informer.go

```diff
@@ -121,6 +128,7 @@ func (s *Service) Boot() {
 		err := backoff.RetryNotify(o, s.backOff, n)
 		if err != nil {
 			s.logger.Log("error", fmt.Sprintf("stop informer boot retries due to too many errors: %#v", microerror.Mask(err)))
+			s.exitFunc(1)
```

Comment:
This is the actual fix. Right now the eventer runs but is actually broken. Now it will fail and the alerting will notice.